packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import UserIconGrey from '../../images/UserIconGrey';
+import BackArrowButton from '../common/BackArrowButton';
+import FormButton from '../common/FormButton';
+import Container from '../common/styled/Container.css';
+import SendIcon from '../svg/SendIcon';
+import NFTTransferModal from './NFTTransferModal';
+
+const StyledContainer = styled(Container)`
+    .container {
+        max-width: 429px;
+        position: relative;
+
+        img {
+            width: 100%;
+            max-width: 429px;
+            margin-bottom: 83px;
+
+            filter: drop-shadow(0px 100px 80px rgba(0, 0, 0, 0.07)) 
+            drop-shadow(0px 41.7776px 33.4221px rgba(0, 0, 0, 0.0503198)) 
+            drop-shadow(0px 22.3363px 17.869px rgba(0, 0, 0, 0.0417275)) 
+            drop-shadow(0px 12.5216px 10.0172px rgba(0, 0, 0, 0.035)) 
+            drop-shadow(0px 6.6501px 5.32008px rgba(0, 0, 0, 0.0282725)) 
+            drop-shadow(0px 2.76726px 2.21381px rgba(0, 0, 0, 0.0196802));
+        }
+
+        .desc {
+            margin-top: 48px;
+            margin-bottom: 56px;
+
+            font-weight: 500;
+            font-size: 16px;
+            line-height: 150%;
+            display: flex;
+            align-items: center;
+            color: #272729;
+        }
+    }
+
+    .owner {
+        p {
+            font-weight: 500;
+            font-size: 12px;
+            line-height: 150%;
+            display: flex;
+            align-items: center;
+            letter-spacing: 0.115em;
+            color: #A2A2A8;
+        }
+
+        .inner {
+            height: 30px;          
+            line-height: 30px; 
+            display: flex;         
+            align-items: center;
+
```

Comment:
```suggestion
            word-break: break-all;
```

Let's add `break-all` so that long account ID's don't overflow the screen width (esp. on mobile)